project-koku/koku
# [COST-3208] AWS HCS gather CCSP data

## Jira Ticket

[COST-3208](https://issues.redhat.com/browse/COST-3208)

## Description

This change will update the azure queries to also search for CCSP data

## Testing

1. Checkout Branch
2. Make sure you have HCS debug on in your env so you don't have to worry about unleash `ENABLE_HCS_DEBUG=True`
3. Restart Koku
4. Send the dev AWS source and you should see a few HCS files that contain multiple line items with one being a Private Offer and one being CCSP data. If you need help with this, reach out to Corey

## Notes

...
